thirdeye/thirdeye-frontend/app/pods/components/contribution-table/component.js

```diff
@@ -123,7 +123,7 @@ export default Ember.Component.extend({
         return index;
       }
     }
-    return dates.length - 1;
+    return dates.length;
```

Comment:
this might result in an undefined behavior. since our array starts at index 0, `dates[dates.length]` will result in an error

Reply:
good catch. I fixed the code.
usually the end index is the first excluded index. I assume this this is true for JS as well.